myhpom/tasks.py

```diff
@@ -57,6 +57,12 @@ def run(self, document_url_id, document_host=None):
         elif response.status_code == 404:
             cf_run.status = CloudFactoryDocumentRun.STATUS_NOTFOUND
             cf_run.save()
+        elif response.status_code == 201:
+            cf_run.status = CloudFactoryDocumentRun.STATUS_PROCESSING
+            cf_run.save()
+        else:
+            cf_run.status = CloudFactoryDocumentRun.STATUS_ERROR
+            cf_run.save()
 
         # if the run could not be created (status_code != 201), send support email
         # -- we need to understand why the the run could not be created at CloudFactory.
```

Comment:
now that I've got all the way out of 'review' mode and into try mode the first thing I stumble on is not setting up my credentials..this block here should save result to response_content here, even though it wasn't the expected response.

Reply:
Another aspect of this - we should have the actual secrets stored in our secrets files (CLOUDFACTORY_LINE_ID, etc al). They aren't currently.

Reply:
The response_content is now saved no matter what during Submit..Run (if there is a response – i.e., not if there is a RequestError, but otherwise).

Reply:
I've added the CLOUDFACTORY_* variables to the ansible-vault templates for staging, production, mmhstaging, and mmhproduction.